packages/core/admin/admin/src/pages/SettingsPage/components/Tokens/LoadingView/index.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import {
+  SettingsPageTitle,
+  LoadingIndicatorPage,
+  useFocusWhenNavigate,
+} from '@strapi/helper-plugin';
+import { HeaderLayout, ContentLayout } from '@strapi/design-system/Layout';
+import { Main } from '@strapi/design-system/Main';
+import { Button } from '@strapi/design-system/Button';
+import Check from '@strapi/icons/Check';
+import { useIntl } from 'react-intl';
+import PropTypes from 'prop-types';
+
+const LoadingView = ({ apiTokenName }) => {
+  const { formatMessage } = useIntl();
+  useFocusWhenNavigate();
+
+  return (
+    <Main aria-busy="true">
+      <SettingsPageTitle name="API Tokens" />
+      <HeaderLayout
+        primaryAction={
+          <Button disabled startIcon={<Check />} type="button" size="L">
+            {formatMessage({ id: 'global.save', defaultMessage: 'Save' })}
+          </Button>
+        }
+        title={
+          apiTokenName ||
+          formatMessage({
+            id: 'Settings.apiTokens.createPage.title',
+            defaultMessage: 'Create API Token',
```

Comment:
This component should be abstracted, right now it's only referring to API Tokens